Zoo.py

```diff
@@ -0,0 +1,100 @@
+# This is a sample Python script.
+import datetime
+
+
+
+class Animal:
+    was_born = False #проверяем было ли рождено животное после инициализации нельзя менять день рождения и цвет
+    birthday = str #Установка дня рождения
+    def get_birthday(self):
+        return self.birthday
+    def set_birthday (self):
+        if self.was_born == True:
+            print("Животное было рождено" + self.get_birthday() + "и с этим уже ничего не поделать")
+        elif self.was_born == False:
+            print("Введите день рождения зверя \n")
+            date = input()
+            self.birthday = date
+    color = str #Установка цвета животного
+    def get_color (self):
+        return self.color
+    def set_color (self):
+        if self.was_born == True:
+            print("Животное было рождено цвета" + self.get_color() + "и с этим уже ничего не поделать")
+        elif self.was_born == False:
+            print("Введите цвет зверя \n")
+            color = input()
+            self.color = color
+    name = str #Установка клички животногоj
+    def get_name (self):
+        return self.name
+    def set_name (self):
+        print("Введите кличку животного")
+        name = input()
+        self.name = name
+
+    def __init__(self):
+        self.set_birthday()
+        self.set_color()
+        self.set_name()
+        self.was_born = True
+
+
+
+
+class Predator(Animal):
+    kill_list = list
+    def kill(self):
+        pass
+class Wolf(Predator):
+    pass
```

Comment:
Two blank line between classes for readability
```suggestion
    pass
    
    
```